public/index.html

```diff
@@ -57,7 +57,7 @@ <h1 class="text-5xl md:text-6xl font-semibold text-orange-800">
     </div>
 
     <div class="leading-5 mb-8">
-        <a href="https://github.com/CuyZ/MagicConstant" class="font-bold text-xl text-blue-600 hover:underline">
+        <a href="https://cuyz.io/WebZ/" class="font-bold text-xl text-blue-600 hover:underline">
```

Comment:
Should it be a link directly to the doc or the repo?

Reply:
Hm I guess the doc is ok since we're already on a website.